packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -73,6 +75,8 @@ export class CallContext {
   private _emitter: EventEmitter;
   private _atomicId: number;
   private _callIdHistory: CallIdHistory = new CallIdHistory();
+  /* @conditional-compile-remove(reaction) */
+  private _timeOutId: { [key: string]: any } = {};
```

Comment:
nit: we are moving away from allowing `any` as a type in the repo. I believe that the type that should be here is `number` 😊

https://developer.mozilla.org/en-US/docs/Web/API/setTimeout

Reply:
Thanks for this pointer Donald. Will update accordingly.